presidio-analyzer/presidio_analyzer/__init__.py

```diff
@@ -14,10 +13,23 @@
 from presidio_analyzer.analyzer_engine import AnalyzerEngine
 from presidio_analyzer.analyzer_request import AnalyzerRequest
 
+# Define default loggers behavior
+
+# 1. presidio_analyzer logger
+
 logging.getLogger("presidio-analyzer").addHandler(logging.NullHandler())
```

Comment:
shouldn't there be a way to change this behavior? will users of the package never want logging or just by default?

Reply:
They can use the same logic as we have in app.py. We can have an if/else here with some constant but I don't know if it'll make it easier to work with. WDYT?

Reply:
if the client of the package can override this behavior as we do in app.py, without changing this code than its ok. if they have to change this code to allow it to log, i think it can be better. approving.

Reply:
Thanks. They would have to write code (toconfigure logging) but not to change existing code. Hope this is what you mean.